common/test/acceptance/pages/lms/discussion.py

```diff
@@ -112,6 +112,16 @@ def is_response_editor_visible(self, response_id):
         """Returns true if the response editor is present, false otherwise"""
         return self._is_element_visible(".response_{} .edit-post-body".format(response_id))
 
+    @wait_for_js
+    def _is_discussion_body_visible(self):
+        return self._is_element_visible(".post-body")
+
+    def _is_mathjax_preview_available(self):
+        return self.q(css=".MathJax_Preview").text[0] == ""
+
+    def _is_mathjax_rendered(self):
```

Comment:
Since all these methods are being used from outside the class, I don't see why they need an '_' at the start.